[2024-11-06T16:30:04.962+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-06T16:30:05.009+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgess_v2.insert_into_postgres_table scheduled__2024-11-05T00:00:00+00:00 [queued]>
[2024-11-06T16:30:05.033+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgess_v2.insert_into_postgres_table scheduled__2024-11-05T00:00:00+00:00 [queued]>
[2024-11-06T16:30:05.035+0000] {taskinstance.py:2866} INFO - Starting attempt 4 of 6
[2024-11-06T16:30:05.084+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): insert_into_postgres_table> on 2024-11-05 00:00:00+00:00
[2024-11-06T16:30:05.127+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=12040) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-06T16:30:05.136+0000] {standard_task_runner.py:72} INFO - Started process 12043 to run task
[2024-11-06T16:30:05.133+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgess_v2', 'insert_into_postgres_table', 'scheduled__2024-11-05T00:00:00+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress.py', '--cfg-path', '/tmp/tmpx8togpsh']
[2024-11-06T16:30:05.142+0000] {standard_task_runner.py:105} INFO - Job 49: Subtask insert_into_postgres_table
[2024-11-06T16:30:05.191+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-06T16:30:05.423+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgess_v2.insert_into_postgres_table scheduled__2024-11-05T00:00:00+00:00 [running]> on host 61aa06b65713
[2024-11-06T16:30:05.428+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-06T16:30:06.896+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Adithya' AIRFLOW_CTX_DAG_ID='dag_with_postgess_v2' AIRFLOW_CTX_TASK_ID='insert_into_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2024-11-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-05T00:00:00+00:00'
[2024-11-06T16:30:06.910+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-06T16:30:06.969+0000] {sql.py:278} INFO - Executing: 
insert into dag_runs(dt,dag_id) values ('2024-11-05','dag_with_postgess_v2')
[2024-11-06T16:30:07.001+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-11-06T16:30:07.041+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-11-06T16:30:07.073+0000] {sql.py:544} INFO - Running statement: 
insert into dag_runs(dt,dag_id) values ('2024-11-05','dag_with_postgess_v2'), parameters: None
[2024-11-06T16:30:07.081+0000] {sql.py:553} INFO - Rows affected: 1
[2024-11-06T16:30:07.149+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-06T16:30:07.150+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgess_v2, task_id=insert_into_postgres_table, run_id=scheduled__2024-11-05T00:00:00+00:00, execution_date=20241105T000000, start_date=20241106T163005, end_date=20241106T163007
[2024-11-06T16:30:07.255+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-06T16:30:07.287+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-06T16:30:07.291+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
